modelutils.py

```diff
@@ -91,3 +108,56 @@ def get_sequential_groups(model):
         ]
     else:
         raise ValueError(MODEL_ERROR_MSG.format(model.config.model_type))
+
+def read_quant_weight_from_file(load_path, block_i, layer_name):
+    return torch.load(load_path + '/' + str(block_i) + '/' + layer_name)
+
+
+def load_quantized_model(model, load_path):
+    layers = get_layers(model)
+    for i in range(len(layers)):
```

Comment:
suggest to use tqdm:
`    for i in trange(len(layers), decs="Loading saved layers"):`